config/proptest-regressions/iroha.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc f4e426272a15348adb1409e1ed846a56fad3f3a671ad67eae9a1ea440dab4088 # shrinks to proxy = ConfigurationProxy { public_key: None, private_key: None, disable_panic_terminal_colors: None, kura: None, sumeragi: None, torii: None, block_sync: None, queue: None, logger: None, genesis: None, wsv: None, network: None, telemetry: None, snapshot: None }
```

Comment:
Did you intend to commit it? (I am not sure how proptest works exactly, so I am not sure if this testcase is meaningful)

Reply:
tnx, this was a leftover from cherry-pick